### ISSUE TITLE
Better support for clojure.lang.PersistentQueue and other core Clojure data structures

### ISSUE THREAD

**Issue by rbeesley** [vlaaad/reveal#24]
Consider the following:
```
  (pop
   (->
    (clojure.lang.PersistentQueue/EMPTY)
    (conj "Item 1")
    (conj "Item 2")))
```

The output in Reveal is:
![image](https://user-images.githubusercontent.com/3703112/105915576-bc398300-5fe4-11eb-86e3-d9728bf7d013.png)

When shown as a table it is something which can be navigated, however this is the output for Cavla:
![image](https://user-images.githubusercontent.com/3703112/105915934-1fc3b080-5fe5-11eb-9ddd-4dbda36e071f.png)

The representation of the data is actually easier to digest in Calva than it is in Reveal without navigating into the object itself. I believe this could be addressed with extending, but it also seems that core Clojure structures should naturally come with a friendlier representation OOTB.

**Comment by vlaaad** [comment]
Hi! Thanks, I think it makes sense to show queue in a more list-like form, maybe `#reveal/persistent-queue(...)` or just `(...)`.

**Comment by vlaaad** [comment]
Fixed in `1.3.194`
